samples/apps/bookinfo/route-rule-ratings-test-v2.yaml

```diff
@@ -0,0 +1,17 @@
+type: route-rule
+spec:
+  name: ratings-test-v2
+  destination: ratings.default.svc.cluster.local
+  precedence: 2
+  route:
+  - tags:
+      version: v2
+---
+type: route-rule
+spec:
+  name: reviews-test-v2
```

Comment:
The reviews version v1 does not query the micro service ratings only version 2 and 3 of reviews services do that.